tools/PI/DevHome.PI/Views/PrimaryWindow.xaml.cs

```diff
@@ -98,6 +101,6 @@ public void HandleHotKey(int keyId)
             TargetAppData.Instance.SetNewAppData(process, hWnd);
         }
 
-        DBarWindow ??= new();

```

Comment:
Super nit: In `ShowBarWindow` we can reduce the indentation if you move `DBarWindow ??= new();` from this method to the beginning of the `ShowBarWindow` method.